### DISCUSSION TITLE
Basset problem on my project

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#1050] [Q&A (Help)] [answered]
I've set my filesystem.php with these disks:
```
    'disks' => [

        'local' => [
            'driver' => 'local',
            'root' => storage_path('app'),
            'throw' => false,
        ],

        'public' => [
            'driver' => 'local',
            'root' => storage_path('app/public'),
            'url' => env('APP_URL').'/storage',
            'visibility' => 'public',
            'throw' => false,
        ],


        'basset' => [
            'driver' => 'local',
            'root' => public_path(''),
            'url' => '/',
            'visibility' => 'public',
            'throw' => false,
        ],

    ],
```
and php artisan basset:check result is:

   INFO  Checking Backpack Basset installation.  

  Initializing basset check ................................................................................................................... DONE  
  Checking cache storage ...................................................................................................................... DONE  
  Fetching a basset ........................................................................................................................... DONE  

But on browser console i've these errors:
<img width="810" alt="Screenshot 2024-06-26 alle 08 37 01" src="https://github.com/Laravel-Backpack/community-forum/assets/1261851/05c56875-d448-41b4-a0f6-3edf7d848e84">

These files are here: vendor/backpack/theme-coreuiv2/resources/assets/css/source-sans-pro.css
Some help to fix it ?

Thanks

**Comment by pxpm** [comment]
Hey @blondie63 

And in network tab you can see where is it trying to get those files from ? 

Cheers

**Reply by blondie63** [reply]
Hi @pxpm 
<img width="822" alt="Screenshot 2024-06-26 alle 10 58 39" src="https://github.com/Laravel-Backpack/community-forum/assets/1261851/c01969a8-4e7c-4c8f-ad8a-3d3feb69a18e">

**Reply by pxpm** [reply, marked as answer]
Interesting, thanks for the report. 

Digging a little bit more I found that the `storage/basset` prefix is hardcoded in the css file. https://github.com/Laravel-Backpack/theme-coreuiv2/blob/10e943a491e1f4ad5fb5e095105dc7a77628f0ec/resources/assets/css/source-sans-pro.css#L14

I guess that was a workaround to load those assets inside css and a proper solution was never researched. The fact that you changed the "default" configuration highlighted this problem. 

I've created a PR to fix it: https://github.com/Laravel-Backpack/theme-coreuiv2/pull/38

You can test that PR by changing your `composer.json` to: `"backpack/theme-coreuiv2": "dev-fix-sans-pro-paths as 1.2.3"`

If it fixes it for you let us know, it would help speeding up the merging process if more people test the changes 🙏 

Cheers

**Reply by blondie63** [reply]
Ok this workaround it's ok but i think it's better i migrate to newer theme..
@pxpm what you think ?

**Reply by pxpm** [reply]
Migrating to a new theme can be time consuming, depending on how much you customized the current theme. 

From CUIv2 to Tabler or CUIv4, the change in bootstrap (from v4 to v5) requires a little bit of attention, since some classes need to be updated. 

If you don't have much customizations it should be fairly easy as just installing the new theme, and changing the config to use the new theme. 

I've just released a new version of core-ui with this fix, you can get it with a composer update. 

Thanks for the report @blondie63 🙏 

Cheers
